internal/services/purview/purview_account_resource_test.go

```diff
@@ -45,6 +45,28 @@ func TestAccPurviewAccount_requiresImport(t *testing.T) {
 	})
 }
 
+func TestAccPurviewAccount_withManagedResourceGroupName(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_purview_account", "test")
+	r := PurviewAccountResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.withManagedResourceGroupName(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
```

Comment:
could this include a check to see if the managed_resource_group_name itself has been set?

Reply:
Moved managed_resource_group_name to parameter and added the check for it.
I removed the update part as well since this is a ForceNew property